content/ios/user.md

```diff
@@ -136,3 +136,8 @@ operation.container = [SKYContainer defaultContainer];
 ## User Relations
 
 See [Friends and Followers]({{< relref "relation.md" >}}).
+
+# Social Login
```

Comment:
I think this part should follow the login section? You may have a look on how the JS guide writes it right now. And some sample code on the iOS SDK will be necessary too. I think a Facebook login plugin exists in the SDK now